app/src/main/java/org/simple/clinic/summary/compose/StatinNudgeView.kt

```diff
@@ -55,54 +63,105 @@ fun StatinNudge(
     Card(
         modifier = modifier
     ) {
-      Column(
-          modifier = Modifier
-              .padding(16.dp),
-          horizontalAlignment = Alignment.CenterHorizontally
+      BoxWithConstraints(
+          modifier = Modifier.padding(16.dp)
       ) {
-        Row {
-          Box(
-              modifier = Modifier
-                  .weight(2f)
+        val constraints = constraints
```

Comment:
Is there a reason why we created a variable called `constraints`, when we can directly access `constraints` in the `BoxWithConstraints` scope?

Reply:
Without creating a variable, it is not accessible in RiskText Which is inside a column. 
Error:- `'val constraints: Constraints' can't be called in this context by implicit receiver. Use the explicit one if necessary`